src/test/java/net/sf/jabref/logic/formatter/bibtexfields/HtmlToUnicodeFormatterTest.java

```diff
@@ -0,0 +1,59 @@
+package net.sf.jabref.logic.formatter.bibtexfields;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class HtmlToUnicodeFormatterTest {
```

Comment:
Add this class to the parametrized FormatterTest class (not sure about the exact name)